docs/views/tutorials-and-examples.html

```diff
@@ -213,9 +213,7 @@ <h2 class="govuk-heading-m">Page templates</h2>
         you prototype realistic journeys connecting your service with GOV.UK content.
       </p>
       <p>
-        Read more about how to get a <a href="https://design-system.service.gov.uk/patterns/start-pages/">
-        Start page</a> or <a href="https://design-system.service.gov.uk/patterns/step-by-step-navigation/">
-        Step by step navigation</a> for your service in the GOV.UK Design System.
+        Read the <a href="https://design-system.service.gov.uk/patterns/start-pages/">start pages guidance</a> and the <a href="https://design-system.service.gov.uk/patterns/step-by-step-navigation/"> step by step navigation guidance</a> to learn how to make these pages live for your service.```
```

Comment:
`"> step`
I think there's an errant space here that would get included and underlided in the link

Reply:
```suggestion
        Read the <a href="https://design-system.service.gov.uk/patterns/start-pages/">start pages guidance</a> and the <a href="https://design-system.service.gov.uk/patterns/step-by-step-navigation/">step by step navigation guidance</a> to learn how to make these pages live for your service.```
```

Reply:
@joelanman Excellent catch, thanks!